### DISCUSSION TITLE
How to run PrivateGPT on SSL?

### DISCUSSION THREAD

**Discussion by fahad-gcx** [imartinez/privateGPT#1567] [Q&A]
Hi

I am right now running privateGPT successfully on a Windows machine. However, by default, it is accessible on a non-SSL URL i.e. http://. 
How can we run the privateGPT on SSL? Or can we embed an SSL certificate on the privaetGPT web interface?

Thanks
FAD

**Comment by tedraymond** [comment]
Add to the uvicorn __main__.py, 

Ssl_key=, ssl_cert=, 

Those will enable https
